### ISSUE TITLE
Can't using Source Han Serif in Acrobat (but can in other apps)

### ISSUE THREAD

**Issue by xoda0** [adobe-fonts/source-han-serif#170]
I installed the Source Han Serif Simplified Chinese font from Adobe. However, when I try to use this font in the header or footer of a document in Acrobat, the text simply disappears. I can use other Chinese fonts (such as the OS included Songti) but not Source Han Serif. Funny enough, I can use Source Han Serif in other Adobe apps (such as Illustrator). But in Acrobat it gives me trouble.

I have already tried reinstalling the font (reactiving it in the Adobe CC app) as well as reinstalling Acrobat to no avail. I also tried downloading the font directly from Github, and also installing the Google variant (Noto Serif SC).

I also tried this on 3 different computers - including 2 Macs (running 13.1) and 1 Windows 11 PC

I still think this issue is related to the font somehow being improperly installed. Any ideas? Thanks!

video of what happens when I use the font in Acrobat: https://www.youtube.com/watch?v=TTOVswnnN0I

**Comment by xoda0** [comment]
Thanks for the reply! It's good to know I'm not the only one with this issue. 

I did some more testing. The issue just seems to be with Source Han Serif - as other fonts work well in Acrobat. In fact, Hans Serif doesn't work in other PDF applications as well, including Foxit. I was able to get it working in PDF Expert on the Mac, but it crashes a lot and is unreliable.

On the other hand, in word processing applications such as Word & Pages (iWork) Source Han Serif works without issue. 

The same applies for the Noto variant.

Given this result, I thought there was some mutual incompatibility between PDFs and Source Han Serif - which is ironic considering Adobe is behind both products
